source/plugins/go/src/telemetry.go

```diff
@@ -302,6 +302,14 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 		osType := os.Getenv("OS_TYPE")
 		if osType != "" && strings.EqualFold(osType, "windows") {
 			telegrafEnabled["IsTelegrafEnabled"] = os.Getenv("TELEMETRY_CUSTOM_PROM_MONITOR_PODS") // If TELEMETRY_CUSTOM_PROM_MONITOR_PODS, then telegraf is enabled
```

Comment:
change the variable name from telegrafEnabled to telegrafConfig since this will hold multiple properties.